src/status_im/contexts/chat/composer/gradients/style.cljs

```diff
@@ -31,13 +31,14 @@
    :position :absolute
    :bottom   0
    :left     0
-   :right    0
-   :z-index  2})
+   :right    0})
 
 (defn bottom-gradient
   []
-  {:colors [(colors/theme-colors colors/white colors/neutral-95)
-            (colors/theme-colors colors/white-opa-0 colors/neutral-95-opa-0)]
-   :start  {:x 0 :y 1}
-   :end    {:x 0 :y 0}
-   :style  bottom-gradient-style})
+  {:colors    [(colors/theme-colors colors/white-opa-90 colors/neutral-95-opa-60)
+               (colors/theme-colors colors/white-opa-30 colors/neutral-95-opa-0)
+              ]
```

Comment:
Hi @clauxx, Thank you very much for your PR.
1. Please can you elaborate, how these changes are related to current issue?
2. Also, Sorry I am having little hard time locating these values. Please can you provide an link?

Reply:
1. It's an additional UI fix to the composer, where for multiline messages there should be a bottom shadow at the bottom when collapsed/minimized. I mentioned it to QA in a comment above, but will add it to the issue description as well.
2. If you mean a design link, then the composer design should do it. But I didn't change the colors here, just adjusted the opacity and the gradient position so it looks according to the design.

Reply:
> so it looks according to the design

Yes, that's what I am curious about. Please can you provide any reference what we are fixing here and what is not matching with design. So we can know why it doesn't look design. Is it implementation issue or we are using different values then figma.

Reply:
Checked it against the designs and seems like this change is not necessary. I remember comparing it with the design at the time and the shadow being barely visible but it looks fine without these changes. Just the z-index fix should be enough to fix the bottom shadow, as before it wasn't visible at all. Thanks for being thorough with this :+1: 

![0](https://github.com/status-im/status-mobile/assets/21865759/cb76f963-54c9-47c0-af30-78a04e2e2a0c)
![0](https://github.com/status-im/status-mobile/assets/21865759/d67f836d-1e0f-4684-8691-318c4dde00ee)